assets/js/modules/analytics-4/utils/withCustomDimensions.js

```diff
@@ -390,6 +428,23 @@ export default function withCustomDimensions( options = {} ) {
 						</MetricTileError>
 					);
 				}
+
+				if ( isGatheringData ) {
+					return (
+						<MetricTileTable
+							infoTooltip={ tileInfoTooltip }
+							moduleSlug="analytics-4"
+							title={ tileTitle }
+							Widget={ Widget }
+							ZeroState={ () =>
```

Comment:
Note that there's no need for the separate `ZeroDataMessage`-like component specced in the IB, as we only need to make this change in one place, and there's no point creating a separate component for a single string.